examples/experimental/onnx/classification/onnx_ptq_classification.py

```diff
@@ -101,9 +97,11 @@ def run(onnx_model_path: str, output_model_path: str,
         "--batch_size", help="Batch size for initialization", type=int, default=1)
     parser.add_argument(
         "--shuffle", help="Whether to shuffle dataset for initialization", default=True)
-    parser.add_argument("--input_shape",
-        help="Model's input shape. e.g. [1, 3, 224, 224]. If it's not given, it automatically infers input shape.",
-                        nargs="+", type=int, default=None)
+    parser.add_argument(
```

Comment:
You are duplicating code here in the segmentation sample. Extract the common parser part like it is done in the [PT](examples/torch/common/argparser.py) and [TF](examples/tensorflow/common/argparser.py) samples.

Reply:
Done